src/overlays/actors/ovl_En_Famos/z_en_famos.c

```diff
@@ -28,133 +62,809 @@ const ActorInit En_Famos_InitVars = {
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_808AE62C = {
-    { COLTYPE_NONE, AT_NONE | AT_TYPE_ENEMY, AC_NONE, OC1_NONE, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK0, { 0xF7CFFFFF, 0x04, 0x08 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_NONE, },
+static ColliderCylinderInit sCylinderInit2 = {
+    {
+        COLTYPE_NONE,
+        AT_NONE | AT_TYPE_ENEMY,
+        AC_NONE,
+        OC1_NONE,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0xF7CFFFFF, 0x04, 0x08 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_NONE,
+    },
     { 70, 10, 0, { 0, 0, 0 } },
 };
 
-// static ColliderJntSphElementInit sJntSphElementsInit[2] = {
-static ColliderJntSphElementInit D_808AE658[2] = {
+// the emblem at the from and the back is one limb running front to back through the torso
+// this is both sides of the emblem, front and back
+static ColliderJntSphElementInit sJntSphElementsInit[2] = {
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00002000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { 2, { { 2500, 0, 0 }, 20 }, 100 },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00002000, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { FAMOS_LIMB_EMBLEM, { { 2500, 0, 0 }, 20 }, 100 },
     },
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00002000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
-        { 2, { { -1500, 0, 0 }, 20 }, 100 },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00002000, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
+        { FAMOS_LIMB_EMBLEM, { { -1500, 0, 0 }, 20 }, 100 },
     },
 };
 
-// static ColliderJntSphInit sJntSphInit = {
-static ColliderJntSphInit D_808AE6A0 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_TYPE_1, COLSHAPE_JNTSPH, },
-    2, D_808AE658, // sJntSphElementsInit,
+static ColliderJntSphInit sJntSphInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_TYPE_1,
+        COLSHAPE_JNTSPH,
+    },
+    2,
+    sJntSphElementsInit,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_808AE6B8[] = {
+typedef enum {
+    /* 0 */ FAMOS_ANIMATED_MAT_NORMAL, // normal is greenish
+    /* 1 */ FAMOS_ANIMATED_MAT_FLIPPED // flipped is orange/yellowish
+} FamosAnimatedMatArrayIndexes;
+
+static AnimatedMaterial* sEmblemAnimatedMats[] = { gFamosNormalGlowingEmblemTexAnim,
+                                                   gFamosFlippedGlowingEmblemTexAnim };
```

Comment:
```suggestion
static AnimatedMaterial* sEmblemAnimatedMats[] = { gFamosNormalGlowingEmblemTexAnim,
                                                   gFamosFlippedGlowingEmblemTexAnim, };
```
Probably formats a little better

Reply:
```
-static AnimatedMaterial* sEmblemAnimatedMats[] = { gFamosNormalGlowingEmblemTexAnim,
-                                                   gFamosFlippedGlowingEmblemTexAnim, };
+static AnimatedMaterial* sEmblemAnimatedMats[] = {
+    gFamosNormalGlowingEmblemTexAnim,
+    gFamosFlippedGlowingEmblemTexAnim,
+};
```
Not really convinced this is better, but don't care enough to not leave as it is now.